MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -1268,7 +1268,7 @@ private static void MergeAlbumArtist(BaseItem source, BaseItem target, bool repl
                 {
                     targetHasAlbumArtist.AlbumArtists = sourceHasAlbumArtist.AlbumArtists;
                 }
-                else if (sourceHasAlbumArtist.AlbumArtists.Count > 0)
+                else
```

Comment:
If the goal here is to call Distinct on the list of target album artists then a third branch should be added to avoid the Concat call